src/include/ipc/header.h

```diff
@@ -289,6 +289,7 @@
  */
 
 #define SOF_IPC_TEST_IPC_FLOOD			SOF_CMD_TYPE(0x001)
+#define SOF_IPC_TEST_MEM_USAGE			SOF_CMD_TYPE(0x002)
```

Comment:
Not sure about the "TEST" prefix. This seems more like a software telemetry interface. The TEST_IPC_FLOOD is a specific service used only for testing, so that seems very different. Both have in common that these are probably only debug builds, so maybe SOF_IPC_DEBUG_ is a more appropriate one.

Reply:
I intended to make this service used only from tests, but maybe you're right, that it should be reclassified as debug service, accessible also from kernel code, maybe debugfs?

Reply:
Yep -debug.